src/main/java/com/amazon/aocagent/helpers/SSMHelper.java

```diff
@@ -0,0 +1,149 @@
+package com.amazon.aocagent.helpers;
+
+import com.amazon.aocagent.enums.GenericConstants;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagement;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder;
+import com.amazonaws.services.simplesystemsmanagement.model.Command;
+import com.amazonaws.services.simplesystemsmanagement.model.CommandPlugin;
+import com.amazonaws.services.simplesystemsmanagement.model.CommandStatus;
+import com.amazonaws.services.simplesystemsmanagement.model.DescribeInstanceInformationRequest;
+import com.amazonaws.services.simplesystemsmanagement.model.InstanceInformation;
+import com.amazonaws.services.simplesystemsmanagement.model.InstanceInformationFilter;
+import com.amazonaws.services.simplesystemsmanagement.model.InvalidInstanceIdException;
+import com.amazonaws.services.simplesystemsmanagement.model.ListCommandInvocationsRequest;
+import com.amazonaws.services.simplesystemsmanagement.model.ListCommandsRequest;
+import com.amazonaws.services.simplesystemsmanagement.model.SendCommandRequest;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+@Log4j2
+public class SSMHelper {
```

Comment:
should we name it to ```SSMService``` like we did for other AWS services?  Service classes will directly interact with AWS resources.